ansible/polipo/files/kitchen.local.yml

```diff
@@ -0,0 +1,52 @@
+<%
+require 'socket'
+
+# @return [String] public IP address of workstation used for egress traffic
+def local_ip
+  @local_ip ||= begin
```

Comment:
Is there an easier way to do this?

It depends on external service, which we do not have control over.